src/js/objects/live.grid.js

```diff
@@ -1,173 +1,496 @@
 import MiraUIObject from "./base.js";
+import * as PIXI from "pixi.js";
+import { createHexColors } from "../lib/pixiUiObject.js";
+import ActiveFrameStore from "../stores/activeFrame.js";
 
-export default class LiveGrid extends MiraUIObject {
-	constructor(stateObj) {
-		super(stateObj);
-		this._typeForTouch = {};
-		this._lastPositionForTouch = {};
+const DIRECTION_MARGIN = 6;
+
+const CellStyle = Object.freeze({
```

Comment:
if we want immitate immutability here, we might wanna use nested Object.freeze!?